content/en/docs/install-setup/installing/boot/how-it-works.md

```diff
@@ -68,11 +69,12 @@ env/
     values.yaml  # tekton specific config
 ```
 
-then you can omit the `prow:` indentation in the `env/prow/values.yaml` file to make the YAML you create/edit smaller and simpler. It also means longer term we can generate JSON schema files for each `env/$app/values.yaml` files so that we can support better editor/IDE integration
+then you can omit the `prow:` indentation in the `env/prow/values.yaml` file to make the YAML you create/edit smaller and simpler.
+It also means longer term we can generate JSON schema files for each `env/$app/values.yaml` files so that we can support better editor/IDE integration.
 
 #### values.tmpl.yaml templates
 
-When using `jx step helm apply` we now allow `values.tmpl.yaml` files to use go/helm templates just like `templates/foo.yaml` files support inside helm charts so that we can generate value/secret strings which can use templating to compose things from smaller secret values. e.g. creating a maven `settings.xml` file or docker `config.json` which includes many user/passwords for different registries.
+When using `jx step helm apply` we now allow `values.tmpl.yaml` files to use Go/Helm templates just like `templates/foo.yaml` files support inside Helm charts so that we can generate value/secret strings which can use templating to compose things from smaller secret values. e.g. creating a maven `settings.xml` file or docker `config.json` which includes many user/passwords for different registries.
```

Comment:
```suggestion
When using `jx step helm apply` we now allow `values.tmpl.yaml` files to use Go/Helm templates just like `templates/foo.yaml` files support inside Helm charts so that we can generate value/secret strings which can use templating to compose things from smaller secret values (for example, creating a maven `settings.xml` file or docker `config.json` which includes many user/passwords for different registries).
```